js_modules/dagit/packages/core/src/instance/BackfillRow.tsx

```diff
@@ -37,6 +37,11 @@ import {
 } from './types/BackfillRow.types';
 import {BackfillTableFragment} from './types/BackfillTable.types';
 
+const NoBackfillStatusQuery = [
+  () => Promise.resolve({data: undefined} as QueryResult<undefined>),
+  {data: undefined, called: true, loading: false} as QueryResult<undefined>,
+] as const;
```

Comment:
This is kind of unusual.

Reply:
Yeah I'm not a fan of this :( I want there to be a case where we never call the per-row function at all, but it didn't seem worth going through and making the `useDelayedRowQuery` and `useQueryRefreshAtInterval` accept an optional query object, so I passed in a no-op query instead.

I'll add a detailed comment here for now :-/